src/Umbraco.Headless.Client.Net/Delivery/ContentDeliveryService.cs

```diff
@@ -117,5 +121,18 @@ public ContentDeliveryService(IHeadlessConfiguration configuration, HttpClient h
         /// Gets the Media part of the Content Delivery API
         /// </summary>
         public IMediaDelivery Media { get; }
+
+        private static RefitSettings CreateRefitSettings(IHeadlessConfiguration configuration, ModelNameResolver modelNameResolver)
+        {
+            return new RefitSettings
+            {
+                ContentSerializer = new JsonContentSerializer(new JsonSerializerSettings
```

Comment:
Should we change this to use System.Text.Json instead of Newtonsoft.Json ?

Reply:
Changing it would be a breaking change, so I think we should wait until we have time to see if there's other parts that needs reworking.